frontend/src/components/shared/ProfileRepoList.vue

```diff
@@ -206,29 +218,43 @@
           message: error.value.msg,
           type: "warning"
         })
-      } else {
+        return null
+      }
+
+      if (perPage === INITIAL_PER_PAGE) {
         targetRef.value = data.value
-        if(targetRef.value?.total > 6 && total === 6){
-          targetRef.value.more = true
-        }else if(total > 6){
-          targetRef.value.more = false
-        }
+        targetRef.value.more = data.value.total > perPage
```

Comment:
here targetRef means:

  const collections = ref([])
  const models = ref([])
  const datasets = ref([])
  const codes = ref([])
  const spaces = ref([])
  
  so I'm not sure why we have this code:
          targetRef.value = data.value

data. value should be an object, right?

maybe, we should update the default value of the ref to be object like:

const collections = ref({})
  const models = ref({})
  const datasets = ref({})
  const codes = ref({})
  const spaces = ref({})

Reply:
done